packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Prop,
+  State,
+} from '@stencil/core';
+import { getSlottedNodes } from '../../utils/utils';
+
+@Component({
+  tag: 'va-breadcrumbs',
+  styleUrl: 'va-breadcrumbs.css',
+  shadow: true,
+})
+export class VaBreadcrumbs {
+  @Element() el: HTMLElement;
+
+  /**
+   * Adds an aria-label attribute to the <nav /> element.
+   */
+  @Prop() label: string = 'Breadcrumb';
+
+  /**
+   * Analytics tracking function(s) will not be called
+   */
+  @Prop() disableAnalytics: boolean;
+
+  /**
+   * Adds a custom id attribute to the UL element
+   */
+  @Prop() listId: string;
+
+  /**
+   * Adds CSS class `.va-nav-breadcrumbs--mobile` to the
+   * NAV element. The mobile breadcrumb will always
+   * be displayed while mobileFirstProp is True.
+   */
+  @Prop() mobileFirstProp: boolean = false;
+
+  /**
+   * Adds a custom id attribute to the NAV element
+   */
+  @Prop() navId: string;
+
+  @State() breadcrumbs: Array<Node>;
+
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  private fireAnalyticsEvent(event: MouseEvent): void {
+    if (!this.disableAnalytics) {
```

Comment:
So I know we do enable analytics on all the other components. Seems like when this was set up as a react component they reversed it for some reason. Was thinking we should change this logic to be enable analytics instead to keep the pattern the same as the other web components

Reply:
Some components have `disableEnalytics`, others have `enableAnalytics`. In general, the ones with `enableAnalytics` have the analytics event off by default, because those components are things like form controls, so there were concerns about PII being sent. Components like this have the event fire by default because those concerns don't apply, but `disableAnalytics` is provided in case teams want to change that default behavior.

Here you can see which of the React component have which default:

![image](https://user-images.githubusercontent.com/2008881/145623788-82eebf86-5f6f-454c-b9f2-b776d406a9bc.png)


There are certain ones like `LoadingIndicator` which have the default behavior of not firing an event just because they add noise.